### ISSUE TITLE
[Subscene][Docker] AttributeError. Exception info: "'RetryingCFSession' object has no attribute 'isChallengeRequest'"

### ISSUE THREAD

**Issue by dausruddin** [morpheus65535/bazarr#1042]
**Describe the bug**
I am trying Bazarr using Docker and Subscene is giving me this error
```
Throttling subscene for 10 minutes, until 20/08/01 23:04, because of: AttributeError. Exception info: "'RetryingCFSession' object has no attribute 'isChallengeRequest'"
```

**To Reproduce**
1. Enable Subscene and execute manual search.
2. Check your log

**Expected behavior**
List of subtitles.

**Screenshots**

**Software (please complete the following information):**
 - Bazarr: 0.9.0.1
 - Radarr version: 3.0.0.3327
 - Sonarr version: N/A
 - OS: Ubuntu 18.04

**Additional context**
I am using [docker](https://hub.docker.com/r/linuxserver/bazarr) currently.

**Comment by A-Alhashem** [comment]
I can confirm this is the exact error I have on my machine as well, and i'm using Windows 10 Pro

**Software:**

- Bazarr: 0.9.0.1
- Radarr version: 0.2.0.1504
- Sonarr version: 2.0.0.5343
- OS: Windows 10 Pro

Additional context
Normal Windows 10, installed using github.

**Comment by morpheus65535** [comment]
Should be fix in 0.9.0.2 in dev branch. Can you test it?

**Comment by A-Alhashem** [comment]
> Should be fix in 0.9.0.2 in dev branch. Can you test it?

On it.
Will test it within the hour.

**Comment by A-Alhashem** [comment]
Can't speak on behalf of OP, but subscene is now working flawlessly on the dev branch ver. 0.9.0.2 for me.
Thank you for the works!

**PS:** Just a _minor_ hiccup when upgrading from master branch (ver. 0.9.0.1) to develop branch (ver. 0.9.0.2), I got locked out of my form login, as in my user/pass combo does not work, had to manually disable the form via config.ini authentication and reset it.
Just wanted to point it out if any run into same situation as I have.

Sorry took long to reply on testing it as I actually got stuck fixing the form login screen myself.
